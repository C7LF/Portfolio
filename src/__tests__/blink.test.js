import React from "react"
import Blink from "../components/blink/blink.component"
import renderer from "react-test-renderer"

describe('blink component', () => {
  test('component should render correctly', () => {
    const tree = renderer
      .create(<Blink text="blinker text" />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
