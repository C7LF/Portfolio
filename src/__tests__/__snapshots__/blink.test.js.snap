// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`blink component component should render correctly 1`] = `
<span
  className="blink"
>
  blinker text
</span>
`;
